examples/bridge-app/bridge-common/bridge-app.matter

```diff
@@ -1664,6 +1673,8 @@ endpoint 0 {
   }
 
   server cluster ThreadNetworkDiagnostics {
+    emits event ConnectionStatus;
```

Comment:
These two events are not implemented in the SDK as all (for any app).

Reply:
I generated the bridge app `./scripts/run_in_build_env.sh "./scripts/build/build_examples.py --target linux-x64-bridge build"` and then I saw in out/linux-x64-bridge/gen/third_party/connectedhomeip/examples/bridge-app/bridge-common/zapgen/zap-generated/endpoint_config.h:

```
...
409   │ // clang-format off
 410   │ #define GENERATED_EVENT_COUNT 10
 411   │ #define GENERATED_EVENTS { \
 412   │   /* Endpoint: 0, Cluster: Access Control (server) */ \
 413   │   /* EventList (index=0) */ \
 414   │   0x00000000, /* AccessControlEntryChanged */ \
 415   │   /* Endpoint: 0, Cluster: Basic Information (server) */ \
 416   │   /* EventList (index=1) */ \
 417   │   0x00000000, /* StartUp */ \
 418   │   0x00000001, /* ShutDown */ \
 419   │   0x00000002, /* Leave */ \
 420   │   /* Endpoint: 0, Cluster: General Diagnostics (server) */ \
 421   │   /* EventList (index=4) */ \
 422   │   0x00000003, /* BootReason */ \
 423   │   /* Endpoint: 0, Cluster: Software Diagnostics (server) */ \
 424   │   /* EventList (index=5) */ \
 425   │   0x00000000, /* SoftwareFault */ \
 426   │   /* Endpoint: 0, Cluster: WiFi Network Diagnostics (server) */ \
 427   │   /* EventList (index=6) */ \
 428   │   0x00000000, /* Disconnection */ \
 429   │   0x00000001, /* AssociationFailure */ \
 430   │   0x00000002, /* ConnectionStatus */ \
 431   │   /* Endpoint: 1, Cluster: Actions (server) */ \
 432   │   /* EventList (index=9) */ \
 433   │   0x00000000, /* StateChanged */ \
 434   │ }
 ```
 
 So at this point I believe matter files correctly represent what zap files are configured to do.
 It would need a followup to actually fix this.

Reply:
I now notice that the config is for "wifi network diagnostics" ... trying to figure out what is going on...

Reply:
Tried:

```
jq '.endpointTypes[].clusters[] | select(.enabled == 1) | select(.side == "server") | {name: .name, events: [.events[]? | select(.included ==1) | .name]}' examples/bridge-app/bridge-common/bridge-app.zap
```

and indeed it says:

```
{
  "name": "Thread Network Diagnostics",
  "events": []
}
{
  "name": "WiFi Network Diagnostics",
  "events": [
    "Disconnection",
    "AssociationFailure",
    "ConnectionStatus"
  ]
}
```

Maybe I did not update to master to #24987 ?

Will merge with master and update.